inst/scripts/sample_ExpressionSet_SummarisedExperiment.R

```diff
@@ -38,7 +38,6 @@ ExpressionSetSample <-
     assayData = assayDataSample,
     phenoData = AnnotatedDataFrame(phenoDataSample),
     featureData = AnnotatedDataFrame(featureDataSample),
-    protocolData = annotatedDataFrameFrom(assayDataSample, byrow = FALSE),
```

Comment:
'protocolData' has been removed since it is an optional element of the ExpressionSetSample and it is not required by FacileDataSet object.